docs/app/pages/getting-started/getting-started.component.html

```diff
@@ -53,79 +47,47 @@ <h3>Installation</h3>
     </tabset>
 
-    <br>
-
     <h3>Applying the Stylesheet</h3>
 
     <div class="row-divider"></div>
 
     <p>The UX Aspects stylesheet can be applied in four different ways as outlined in the following tabs:</p>
 
-    <br>
-
     <tabset>
-        <tab heading="CLI">
-            <br>
-            <ul>
-                <li>If using the CLI, the UX Aspects stylesheet can be added in the <code>angular-cli.json</code> file:
-                    <br><br>
-                    <uxd-snippet [code]="cliJson" language="javascript"></uxd-snippet>
-                </li>
-            </ul>
+        <tab heading="Angular CLI">
+            <p>If using the CLI, the UX Aspects stylesheet can be added in the <code>angular-cli.json</code> file:</p>     
+            <uxd-snippet [code]="cliJson" language="javascript"></uxd-snippet>
         </tab>
-        <tab heading="Regular Webpack">
-            <br>
-            <ul>
-                <li>When using regular webpack, UX Aspects can be included in a stylesheet by using:
-                    <br><br>
-                    <uxd-snippet [code]="stylesheetCss" language="css"></uxd-snippet>
-                </li>
-            </ul>
+        <tab heading="Webpack">
+            <p>When using Webpack the UX Aspects stylesheet can be included as follows:</p>          
+            <uxd-snippet [code]="stylesheetCss" language="css"></uxd-snippet>
         </tab>
-        <tab heading="JavaScript/Typescript">
-            <br>
-            <ul>
-                <li>The UX Aspects stylesheet can also be included directly from JavaScript/TypeScript by using webpack:
-                    <br><br>
-                    <uxd-snippet [code]="webpackJs" language="javascript"></uxd-snippet>
-                </li>
-            </ul>
+        <tab heading="JavaScript/TypeScript">
+            <p>The UX Aspects stylesheet can be included directly from JavaScript/TypeScript by using Webpack:</p>
+            <uxd-snippet [code]="webpackJs" language="javascript"></uxd-snippet>
         </tab>
         <tab heading="Non-Angular Sites">
-            <br>
-            <ul>
-                <li>The UX Aspects stylesheet can be applied by adding it to the header of the site as follows:
-                    <br><br>
-                    <uxd-snippet [code]="headerHtml" language="html"></uxd-snippet>
-                    <p>Please note this is the only required step for non-Angular sites. However, this method is not recommended with
-                        the use of <code>node_modules</code>.</p>
-                </li>
-            </ul>
+            <p>If your application does not use Node or Webpack, you can copy the UX Aspects stylesheet into your site's assets 
+                directory from the following location and include it in the head of your HTML:</p>
+            <uxd-snippet [code]="headerHtml" language="html"></uxd-snippet>
+            <p>Please note this is the only required step for non-Angular sites.</p>
```

Comment:
There was a rule about not saying "please" in documentation a while back, so let's drop this now.